packages/integration-tests/tests/applications.test.ts

```diff
@@ -19,35 +24,29 @@ describe('admin console application', () => {
   });
 
   it('should create application successfully', async () => {
-    const application = await authedAdminApi
-      .post('applications', {
-        json: { name: testApplication.name, type: testApplication.type },
-      })
-      .json<Application>();
+    const applicationName = 'test-create-app';
+    const applicationType = ApplicationType.SPA;
 
-    expect(application.name).toBe(testApplication.name);
-    expect(application.type).toBe(testApplication.type);
+    const application = await createApplication(applicationName, applicationType);
 
-    // eslint-disable-next-line @silverhand/fp/no-mutation
-    testApplication.id = application.id;
+    expect(application.name).toBe(applicationName);
+    expect(application.type).toBe(applicationType);
```

Comment:
```suggestion
    expect(application.name).toEqual(applicationName);
    expect(application.type).toEqual(applicationType);
```

Reply:
`toEqual` is used when you want to check that two objects have the same value, so we should use `toBe` here.